projects/rdc/include/rdc_lib/impl/RdcNotificationImpl.h

```diff
@@ -49,6 +74,7 @@ class RdcNotificationImpl : public RdcNotification {
  private:
   std::map<uint32_t, uint64_t> gpu_evnt_notif_masks_;
```

Comment:
If the order is not important here, `std::unordered_map<>` will be faster with `O(1)` for most operations.